lib/core/src/display/shape/glyph/pen.rs

```diff
@@ -0,0 +1,121 @@
+//! The pen is a font-specific term (see
```

Comment:
`The pen is a font-specific term` - lets not use such docs. Expalin with 1-2 sentecnces what it is and then use the link for more info.